modules/raw-rsa-keyring-browser/src/raw_rsa_keyring_web_crypto.ts

```diff
@@ -193,34 +194,46 @@ export class RawRsaKeyringWebCrypto extends KeyringWebCrypto {
     const { wrappingAlgorithm, format, key } = getImportOptions(publicKey)
     const backend = await getWebCryptoBackend()
     const subtle = getNonZeroByteBackend(backend)
+    // @ts-ignore // see TS2769 Note below
```

Comment:
I suggest that rather than ignoring we do something like this

```typescript
async function ImportKeyTypeOverload(
  op: ReturnType<typeof getImportOptions>,
  subtle: SubtleCrypto,
  keyUsages: KeyUsage[]
): Promise<AwsEsdkJsCryptoKey> {
  return op.format == 'jwk'
    ? subtle.importKey(
        op.format,
        op.key,
        op.wrappingAlgorithm,
        false,
        keyUsages
      )
    : subtle.importKey(
        op.format,
        op.key,
        op.wrappingAlgorithm,
        false,
        keyUsages
      )
}
```

I'm going to push a change @texastony , let me know what you think :)